### DISCUSSION TITLE
Donating Funds to Ava

### DISCUSSION THREAD

**Discussion by Lfada** [avajs/ava#3304] [Ideas]
Hi Ava maintainers. I'm Lucas, a core contributor to the Drips project (https://www.drips.network/). I help recipients of donations through their learning and onboarding process.

[OpenZeppelin](https://www.openzeppelin.com/) has been using https://github.com/avajs/ava. It's one of their most critical dependencies!

OpenZeppelin core contributors appreciate the work you all do and want to make sure that you are all well-funded to keep doing it. For that reason, they have decided to donate some funds to your project through [Drips](https://www.drips.network/).

Who can I talk to about it?

cc: @marchuffnagle @sindresorhus @jamestalmage @forresst

**Comment by novemberborn** [comment]
Interesting approach.

From what I can glean from the docs, these funds would go to maintainers personally. Which, fine, but for me personally unless the amount is truly significant not worth the tax hassle.

Our OpenCollective is managed independently which means it's great for expenses like the domain name.

**Reply by Lfada** [reply]
Hey @novemberborn.

Here's OpenZeppelin's [full list of dependencies](https://www.drips.network/app/projects/github/OpenZeppelin/openzeppelin-contracts), including yours. Remember that the amounts you see are the ones that have already been streamed, they will continue to increase. 

[This](https://www.drips.network/app/projects/github/avajs/ava?exact) is Ava's Drips account.

What projects do is to set up a shared wallet (could be a Safe multi-sig), or add each maintainer/contributor's wallet to split the donations. If you don't want to be implicated tax-wise, you could look into splitting 100% of those funds to other dependencies Ava has. Check our [docs](https://docs.drips.network/claim-your-repository), and search for "splits" to see if this option would be interesting for Ava.

**Reply by Lfada** [reply]
hi @novemberborn I Wanted to send this again to see if my response above made sense. 

We would love those funds to be claimed, this way its going to your team, or if Ava doesn't need 100% of the funds, those can be split to your critical dependencies.
